dev-perl/Ace/Ace-1.920.0-r6.ebuild

```diff
@@ -55,7 +54,7 @@ src_compile() {
 		# Parallel compile breaks :(
 		"-j1"
 	)
-	if use elibc_glibc || use elibc_musl || use elibc_uclibc ; then
+	if use elibc_glibc || use elibc_musl ; then
 		mymake+=( "LIBS=-ltirpc -lm" )
```

Comment:
Not sure that -lm belongs there and would ideally use pkg-config but let's not mess with it for now given how Pelr stuff is.